.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: node_js
 node_js:
+  - "5"
+  - "4"
   - "node"
+before_install: npm install -g grunt-cli
```

Comment:
This should probably be removed and added to the package.json file as a devDependency